### ISSUE TITLE
Opt out of SVG optimisation

### ISSUE THREAD

**Issue by daithimaolain** [prismicio/prismic-next#76]
The built-in loader for PrismicNextImage produces a srcset on the image tag for svg images. This behaviour seems to be redundant. Should you rather opt in to this much the same way as it [works](https://nextjs.org/docs/pages/api-reference/components/image#dangerouslyallowsvg) for the underlying Image component.

**Comment by angeloashmore** [comment]
Good idea! This should be doable, though I'm not sure we can respect the global `images.dangerouslyAllowSVG` config value. We'll leave this request open until we are able to work on and publish the feature.
